draft-ietf-quic-transport.md

```diff
@@ -884,6 +884,55 @@ provide an interface to QUIC to tell it about its buffering limits so that there
 is not excessive buffering at multiple layers.
 
 
+## Flow Control Deadlocks {#flow-control-deadlocks}
+
+Flow control provides a means of managing access to the limited buffers
+endpoints have for incoming data.  This mechanism limits the amount of data that
+can be in buffers in endpoints or in transit on the network.  However, there are
+several ways in which limits can produce conditions that can cause a connection
+to either perform suboptimally or deadlock.
+
+Implementations need to be aware of the possibility of deadlocking and that some
+conditions that lead to deadlocking cannot be completely avoided in a protocol
+design.
```

Comment:
I don't think this is accurate. The deadlock problem is not within a protocol here, but in how two protocol implementations (HTTP and QUIC) might interact.

Reply:
My point is that this is not a matter for protocol design, because no protocol can defend against this effectively in its design and that implementations need to handle the issue.  I'll reword.

... and note that this just adds more weight to moving this to applicability :)